frontend/src/app/features/grade-level-selection/grade-level-selection.component.scss

```diff
@@ -9,7 +9,7 @@
     margin-left: 0.25em;
   }
   &.selected {
-    background-color: green;
+    background-color: httat-theme(primary); //$blue;
```

Comment:
I thought we were going with `var(--override--selected-bg-color)` here and on the assistance selection component?

Reply:
You are correct. I added those in.